### PR TITLE
Feature/clubs

### PR DESCRIPTION
**Pull request by MadhushaPrasad**
Capture

These missed elements were added for mobile responsive

### CODE REVIEW

File: components/Navbar.tsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ const Navbar = (): JSX.Element => {
                 </Link>
               </li>
               <li className="hidden lg:block font-medium text-lg hover:text-gray-default transition ease-in">
-                <Link href="/contact">
+                <Link href="/contactUs">
```
Comment (sithumonline) [inline components/Navbar.tsx]: Uppercase for URL ??

File: pages/clubs.tsx [hunk 2]
```diff
@@ -1,32 +1,44 @@
...
-import mozilla from '../public/clubs/mozilla.png'
-import ms from '../public/clubs/ms.png'
-import sliit from '../public/clubs/sliit.png'
-import wif from '../public/clubs/wif.png'
+import foss from "../public/clubs/foss.png"
+import mozilla from "../public/clubs/mozilla.png"
+import ms from "../public/clubs/ms.png"
+import sliit from "../public/clubs/sliit.png"
+import wif from "../public/clubs/wif.png"
+import { HiArrowRight } from "react-icons/hi"
```
Comment (sithumonline) [inline pages/clubs.tsx]: `" "` for imports ??

File: tailwind.config.js [hunk 3]
```diff
@@ -7,32 +7,33 @@ module.exports = {
...
+      textBlackSecondary: "#3D3D3D",
+      textBlackTertiary: "#4F4F4F",
+      white: "#ffffff",
+      blue: "#005BDC",
+      lightBlue: "#21B3D0",
+      lightBlueAccent: "#66a1ff",
+      redAccent: "#ff5454",
+      bgWhite: "#F4F3F5",
+      gradientBlue: "#21B3D0",
+      gradientPurple: "#7B50A2",
```
Comment (sithumonline) [inline tailwind.config.js]: `" "` ??

**Review by sithumonline — CHANGES_REQUESTED** [review]

File: components/contactForm.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+      <Container>
+        <div className="bg-redAccent w-full">
+          <h1>Madhusha</h1>
+        </div>
+      </Container>
+    </div>
+  )
+}
+
+export default ContactForm
```
Comment (chamodshehanka) [inline components/contactForm.tsx]: make the file name start with capital

**Review by chamodshehanka — CHANGES_REQUESTED** [review]

**Review by chamodshehanka — APPROVED** [review]

**Pull request merged**